### ISSUE TITLE
MARACOOS CBEFS 3D Water Temp Layer - Vertical Depth Slider is Flipped

### ISSUE THREAD

**Issue by jamesdoyle21** [asascience/ott-coops#458]
This layer is seeing something similar to #457 however, the opposite is happening. The profile plot looks correct but the spatial layer and timeslider values look wrong. Could this be that the 3D depth slider is flipped? Water should be warmer at the surface in the summer as the profile plot shows.

From Sheri "1.	In OceansMap 3: This one is easiest to test in July 2021 (i.e., a summer month). Time selected is 07/28/2021 at 12:00 am MST, Virtual Buoy @ 37.1763N, 75.3905W. The vertical profile looks correct. However, the values on the left side of the drawer are likely incorrect, as I get a value of 15.62 degC at surface water (green color, corresponds with value correctly), but at the bottom water, the value is higher at 26.64 degC (orange color, corresponds with the value correctly). Temperature should be lower at depth, which would be consistent with the vertical profile."

Surface:
![image](https://user-images.githubusercontent.com/82955395/167194022-002883de-5225-43ea-9466-2f6cb5bd68f5.png)

Depth:
![image](https://user-images.githubusercontent.com/82955395/167194250-bcb3b9a9-fb0d-49a2-b38b-923be7fee12e.png)

**Comment by jamesdoyle21** [comment]
Same thing is happening for MARACOOS CBEFS 3D pH:

Surface:
![image](https://user-images.githubusercontent.com/82955395/167199885-511fd6a9-32a8-4c15-abee-6271d97ecdb0.png)

Depth:
![image](https://user-images.githubusercontent.com/82955395/167199904-f307fdf7-cc81-4587-bdab-7fc1ad8061aa.png)

**Comment by jamesdoyle21** [comment]
Same thing for MARACOOS CBEFS 3D Oxygen

![image](https://user-images.githubusercontent.com/82955395/167201881-b2bb1193-e391-4b8d-80ca-6b71140676e7.png)

**Comment by jamesdoyle21** [comment]
This is also broken in OM1

Surface:
![image](https://user-images.githubusercontent.com/82955395/168647364-75f7b32f-2c77-4890-92ff-4a3fc6ce7c6e.png)


Bottom:
![image](https://user-images.githubusercontent.com/82955395/168647314-1046865e-15d0-4e78-8281-129c7d515bf1.png)

**Comment by kvmartin001** [comment]
@jamesdoyle21 Yes, if he can get the GetFeatureInfo request to send back "sigma" as the depth unit for these layers, all these issues might be fixed instantly. I can confirm it on the frontend once that's done.

**Comment by jamesdoyle21** [comment]
@CoryMartinRPS can you take a look at this issue and comment thread and see if you can assist here?

**Comment by CoryMartinRPS** [comment]
I'm not sure that just returning sigma as the depth unit is the right answer. There's definitely some confusion with the GetCap saying "sigma" and the GFI saying "m", but the way it currently works is very useful. While other sources store data at specific depths that are the same across the dataset, here the depth is calculated on a point-by-point basis. This source uses 20 sigma layers, so for example if the water is 20m deep, there will be a data point around once per meter, but if the water is 5m deep there will be four data points per meter. If EDS were to return the sigmas for elevavtion=all instead of meters, that's new information being lost in favor of already known information.

As noted in #457 I am working on fixing the ordering so "elevation=all" requests will return points in the correct order.

The EDS bug at hand, is that when a single elevation is requested from a 3d dataset, it doesn't always return data from the correct depth depending on whether the 0th depth is surface or bottom. I have a number of ways I could fix things on my end. What I most want to do is to use DepthDirection (which is returned as a field in a GetCap request). Currently, every item in our datacatalog has it set as "down" which is incorrect for many of the 3d datasets. If I were to change that to "up" for datasets that index depth differently, would that impact OceansMap? If not, it would only be used as a flag internal to EDS to make sure the correct depth is used, and you don't have to change how you're making requests at different elevations. If it would, let me know how so I can work with/around what you guys are doing.
